tests/unittests/Makefile

```diff
@@ -56,11 +64,11 @@ $(OUTDIR)/ReferenceFoundation_%.o : $(OUTDIR)/ReferenceFoundation_TestUtils.o $(
 	mkdir -p $(OUTDIR)
 	$(CLANG) -c -o $@ $(CFLAGS) $(word 2,$^) 
 
-.PHONY: clean Shims
```

Comment:
nah, since it's not a rule. having it being a phony rule made make think that `Foundation.UnitTests.exe` was out of date all the time.

---
In reply to: [72871776](https://github.com/Microsoft/WinObjC/pull/726#discussion_r72871776) [](ancestors = 72871776)